docker/storage/cassandra/install.sh

```diff
@@ -16,7 +16,7 @@
 
 set -eu
 
-echo "*** Installing Python and curl"
+echo "*** Temporarily installing Python and curl"
```

Comment:
just now noticed later we remove both of these regardless of if we installed them!